OpenAnatomyExport/OpenAnatomyExport.py

```diff
@@ -380,6 +382,40 @@ def addModelsToRenderer(self, shFolderItemId, boostGouraudColor=False):
             gltfMeshNodeIndex = len(self._gltfNodes)
             self._gltfNodes.append({'mesh': gltfMeshIndex, 'name': meshName})
             gltfFolderNodeChildren.append(gltfMeshNodeIndex)
+          
+        if dataNode and dataNode.IsA("vtkMRMLMarkupsPlaneNode"):
+          inputModelNode = self.createPlaneModelFromMarkupsPlane(dataNode)
+          meshName = dataNode.GetName()
+          self._numberOfProcessedModels += 1
+          self.addLog("Model {0}/{1}: {2}".format(self._numberOfProcessedModels, self._numberOfExpectedModels, meshName))
+
+          # Reuse existing model node if already exists
```

Comment:
This is a too much copy-paste to maintain, please refactor to avoid duplication.